### PR TITLE
Propagate error/success codes back up to wrapper

### PR DESCRIPTION
**Pull request by SGM3**
Untested changes, but this was I was thinking would be the changes. This way I can do stuff like this:
<pre>
gw test && gw deploy
</pre>
And be assure the deploy task only executes if the test  task is successful.

### CODE REVIEW

**Comment by SGM3** [comment]
Tested it locally, seems to work as I intended.

Validated with:
<pre>
> gw -version && echo hello
{..version info...}
hello
>
> gw -badStringStuff && echo hello
{...error message...}
(no hello)
</pre>

File: src/main/java/com/liferay/gw/GradleWrapper.java [hunk 1]
```diff
@@ -7,13 +7,19 @@
...
 		try {
-			gradleExecutor.execute(args);
+			exitCode = gradleExecutor.execute(args);
 		}
 		catch (Exception e) {
 			System.err.println("Error executing gradle wrapper");
 			
 			e.printStackTrace();
+
+			exitCode = -1;//error state?
```
Comment (SGM3) [inline src/main/java/com/liferay/gw/GradleWrapper.java]: Not familiar with the Unix error codes standard, but i think -1 is any generic error

**Comment by david-truong** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by SGM3** [comment]
Rebased

File: src/main/java/com/liferay/gw/GradleWrapper.java [hunk 2]
```diff
@@ -25,14 +25,20 @@
 	public static void main(String[] args) {
 		WrapperExecutor wrapperExecutor = new WrapperExecutor("gradlew");
 
+		int exitCode = 0;
```
Comment (gamerson) [inline src/main/java/com/liferay/gw/GradleWrapper.java:28]: We would need to do this same logic for MavenWrapper

**Comment by SGM3** [comment]
Added same logic to `MavenWrapper`

**Pull request merged**